uncode_scripts/uncode_database_backup

```diff
@@ -1,8 +1,28 @@
 #!/bin/bash
 set -e
 
+function delete_old_backups(){
+  #Deleting backups older than "onepermonth" months
+  mlast=$(date --date="-${onepermonth} months"  '+%Y-%m-%d')
+  for bk in $(gdrive list --no-header --order createdTime -q "createdTime < \"${mlast}\" and trashed = false and 'me' in owners and  name contains 'bk' and name contains 'tar.gz'" | cut -d ' ' -f1)
+  do
+    gdrive delete $bk
+  done
+
+  #Deleting backups older than "ndays" days saving one per month
+  dlast=$(date --date="-${ndays} day"  '+%Y-%m-%d')
+  for bk in $(gdrive list --no-header --order createdTime -q "createdTime < \"${dlast}\" and trashed = false and 'me' in owners and  name contains 'bk' and name contains 'tar.gz'" | awk '{split($(NF-1),d,"-");print $1","d[3]}')
+  do
+    idbk=$(echo $bk | cut -d ',' -f1)
+    day=$(echo $bk | cut -d ',' -f2)
+    if [ "$day" -gt 8 ]; then
```

Comment:
The logic is easier and I think there is no significant difference between the backup of the last week of the month and the first of the next month. But could be other interval like 21 to 28 rather than 1 to 7.

Reply:
I agree with Cristian. I'd prefer to keep the last backup of each month.

Reply:
Any update on this? @andrescorso

Reply:
I updated the code to save the last backup of each month